Analysis/DataModel/src/aodDataModelGraph.cxx

```diff
@@ -67,14 +67,25 @@ enum StyleType : int {
 };
 
 static std::vector<std::pair<std::string, StyleType>> tableStyles = {
-  {"HfTrackIndexProng", StyleType::GREEN},
+  {"HfTrackIndexProng", StyleType::BLUE},
   {"pidResp", StyleType::GREEN},
   {"Mults", StyleType::GREEN},
   {"Cents", StyleType::GREEN},
+  {"Timestamps", StyleType::GREEN},
   {"Jet", StyleType::BLUE},
   {"Mc", StyleType::RED},
   {"V0Datas", StyleType::GREEN},
-  {"CascData", StyleType::GREEN}};
+  {"CascData", StyleType::GREEN},
+  {"TrackSelection", StyleType::GREEN},
+  {"TracksExtended", StyleType::GREEN},
+  {"TracksExtension", StyleType::GREEN},
```

Comment:
These are substrings to find in table names, you can just add one entry with `Extension` so that all extensions will be marked with the chosen color.

Reply:
Thanks. Adjusted